py/desiutil/test/test_bitmask.py

```diff
@@ -6,8 +6,9 @@
                         print_function, unicode_literals)
 # The line above will help with 2to3 support.
 import unittest
-from ..bitmask import BitMask, _MaskBit
+from desiutil.bitmask import BitMask, _MaskBit
```

Comment:
Thanks for the explanation.  I verified that this is a real world problem by trying to test a fresh git clone that wasn't in my PYTHONPATH while an older reference version was.  It picked up the wrong version as you said.

I reverted the import syntax, but I'd appreciate hearing the "other ways to do this".  Do you know a convenient one-liner for running only a single test file instead of all of them, while using the `from ..bitmask import BitMask` syntax?